be/src/vec/functions/array/function_array_aggregation.cpp

```diff
@@ -147,7 +147,13 @@ struct ArrayAggregateImpl {
         const DataTypeArray* data_type_array =
                 static_cast<const DataTypeArray*>(remove_nullable(arguments[0]).get());
         auto function = Function::create(data_type_array->get_nested_type());
-        return function->get_return_type();
+        if (function) {
+            return function->get_return_type();
+        } else {
+            throw doris::Exception(ErrorCode::INVALID_ARGUMENT,
```

Comment:
why not support array_max/array_min with comlex type？

Reply:
Now MIN, MAX, SUM, AVERAGE, PRODUCT agg has not support in be except this datatype: execute_type<UInt8>(res, type, data, offsets) ||
            execute_type<Int8>(res, type, data, offsets) ||
            execute_type<Int16>(res, type, data, offsets) ||
            execute_type<Int32>(res, type, data, offsets) ||
            execute_type<Int64>(res, type, data, offsets) ||
            execute_type<Int128>(res, type, data, offsets) ||
            execute_type<Float32>(res, type, data, offsets) ||
            execute_type<Float64>(res, type, data, offsets) ||
            execute_type<Decimal32>(res, type, data, offsets) ||
            execute_type<Decimal64>(res, type, data, offsets) ||
            execute_type<Decimal128V2>(res, type, data, offsets) ||
            execute_type<Decimal128V3>(res, type, data, offsets) ||
            execute_type<Date>(res, type, data, offsets) ||
            execute_type<DateTime>(res, type, data, offsets) ||
            execute_type<DateV2>(res, type, data, offsets) ||
            execute_type<DateTimeV2>(res, type, data, offsets))